src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -53,7 +53,8 @@ class ExplorerButtonGroup extends React.Component {
         });
     }
     if (nextProps.totalCount !== this.props.totalCount) {
-      this.refreshManifestEntryCount();
+      this.refreshManifestEntryCount().catch(() => {
```

Comment:
what is the point of the catch?

Reply:
There was an error when the component is loading at the first time.

Actually found a right way to stop the error from happening, will remove this `catch`.